lib/Github/Api/Apps.php

```diff
@@ -5,28 +5,14 @@
 use Github\Api\AcceptHeaderTrait;
 
 /**
- * @link   https://developer.github.com/v3/integrations/
+ * @link   https://developer.github.com/v3/apps/
  * @author Nils Adermann <naderman@naderman.de>
  */
-class Integrations extends AbstractApi
+class Apps extends AbstractApi
```

Comment:
I know, I wanted to do a same sort of change but @Nyholm came up with a good point.